### DISCUSSION TITLE
Version Check

### DISCUSSION THREAD

**Discussion by TunayAdaKaracan** [SafeExamBrowser/seb-win-refactoring#677] [General]
I am trying to take a test but it says version mismatch (Not on start, everything starts fine but the page i see says it in browser). I think it is a fork of SEB?. But how is it even possible? I installed everything twice and still not running.

**Comment by dbuechel** [comment]
If the version check of e.g. Moodle fails, then you're using the wrong version. Simply inquire which version you'll need to use and then download it from our official website (or the place where your institution tells you to download SEB from).
